FirstProject/.idea/FirstProject.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
![idea](https://user-images.githubusercontent.com/31060543/29554605-6f66befa-8728-11e7-8ee4-181f6b8bc0b1.png)

Also, idea folder is in gitignore